src/web/components/SlotBodyEnd.tsx

```diff
@@ -16,60 +22,77 @@ type Props = {
-        },
-        targeting: {
-            contentType,
-            sectionName,
-            shouldHideReaderRevenue,
-            isMinuteArticle,
-            isPaidContent,
-            tags,
-        },
-    };
+    const [epicContent, setEpicContent] = useState<{ html: string; css: string}>();
 
-    const endpointUrl = 'https://contributions.guardianapis.com/epic';
-    const { data: responseBody, error } = useApi(endpointUrl, {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json',
-        },
-        body: JSON.stringify(contributionsPayload),
-    });
+    useEffect(() => {
+        const callFetch = async () => {
+            // Putting together the request payload
+            const contributionsPayload = {
+                tracking: {
+                    ophanPageId: window?.guardian.config.ophan.pageViewId,
+                    ophanComponentId: 'ACQUISITIONS_EPIC',
+                    platformId: 'GUARDIAN_WEB',
+                    campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
+                    abTestName: 'remote_epic_test',
+                    abTestVariant: 'api',
+                    referrerUrl: window?.location.origin + window?.location.pathname
+                },
+                localisation: {
+                    countryCode
+                },
+                targeting: {
+                    contentType,
+                    sectionName,
+                    shouldHideReaderRevenue,
+                    isMinuteArticle,
+                    isPaidContent,
+                    tags,
+                    showSupportMessaging: shouldShowSupportMessaging(),
+                    isRecurringContributor: isRecurringContributor(isSignedIn),
+                    lastOneOffContributionDate: getLastOneOffContributionDate(),
+                },
+            };
 
-    if (error) {
-        window.guardian.modules.sentry.reportError(error, 'slot-body-end');
-        return null;
-    }
-
-    if (responseBody && responseBody.data) {
-        const { data } = responseBody;
+            const getEpicContent = () => {
+                const endpointUrl = 'https://contributions.guardianapis.com/epic';
+                return fetch(endpointUrl, {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/json',
+                    },
+                    body: JSON.stringify(contributionsPayload)
+                })
+                .then((response) => {
+                    if (!response.ok) {
+                        const apiError = new Error(response.statusText);
+                        window.guardian.modules.sentry.reportError(apiError, 'slot-body-end');
+                    }
+                    return response;
+                })
+                .then(response => response.json())
+            }
+            const epicResponse = await getEpicContent();
+            setEpicContent(epicResponse.data);
```

Comment:
What does `epicResponse` get set to if `response.ok` is falsey? Do we need to guard against something here?

Reply:
We've refactored this to make use of `useApi` and should be always rendering the Epic when we get a successful response from the API. Errors will be sent to Sentry.